app/assets/javascripts/mobile_cke.js

```diff
@@ -0,0 +1,37 @@
+var onReadyMobileCke = function() {
+  var MEDIUM_BREAKPOINT = 640;
+
+  var ui = null;
+  var focused_editor = null;
+  var modal_editor = CKEDITOR.instances.modal_editor;
```

Comment:
You might want to do this `var modal_editor = CKEDITOR && CKEDITOR.instances && CKEDITOR.instances.modal_editor;` to make sure CKEDITOR is loaded onto the page.